src/routers/handlers/yourCompanies/manageAuthorisedPeopleHandler.ts

```diff
@@ -23,25 +23,23 @@ export class ManageAuthorisedPeopleHandler extends GenericHandler {
         const lang = getTranslationsForView(req.t, constants.MANAGE_AUTHORISED_PEOPLE_PAGE);
         this.viewData = this.getViewData(companyNumber, lang);
         const cancellation: Cancellation = getExtraData(req.session, constants.CANCEL_PERSON);
-        const removal: Removal = getExtraData(req.session, constants.REMOVE_PERSON);
-        let companyAssociations: Associations = await getCompanyAssociations(req, companyNumber);
 
+        let companyAssociations: Associations = await getCompanyAssociations(req, companyNumber);
+        const companyName: string = companyAssociations.items[0].companyName;
```

Comment:
Do we always have at least 1 item?  If `items` is empty we might get an error here